.github/workflows/website-validation.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - uses: pnpm/action-setup@fe02b34f77f8bc703788d5817da081398fad5dd2 # ratchet:pnpm/action-setup@v4
       - uses: actions/setup-node@1a4442cacd436585916779262731d5b162bc6ec7 # ratchet:actions/setup-node@v3
         with:
-          node-version: "18"
+          node-version: "20"
```

Comment:
This update was needed to be compatible with dependency updates. The rest of our repo is on node20, and this is for internal validation of our website build, so we have no customer compat commitments to accommodate here.